src/Sylius/Component/Shipping/Resolver/CompositeMethodsResolver.php

```diff
@@ -45,4 +45,12 @@ public function getSupportedMethods(ShippingSubjectInterface $shippingSubject)
 
         return [];
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supports(ShippingSubjectInterface $subject)
+    {
+        return true;
```

Comment:
Anyway, it seems like this method is not used anywhere except `CompositeMethodsResolver::getShippingMethods()` and always returns true. We can remove it and then that method will return first not-empty result from nested methods resolver.